Lib/test/test_xml_etree.py

```diff
@@ -608,6 +619,7 @@ def test_iterparse(self):
                 iterparse(SIMPLE_XMLFILE, events)
             self.assertEqual(str(cm.exception), "unknown event 'bogus'")
             del cm
+            gc_collect()
```

Comment:
The added `gc_collect()` calls are redundant because the `check_no_resource_warning()` context manager already calls `gc_collect()`.

Reply:
Nice. The initial version of my patch predates `check_no_resource_warning()`.